docker/scripts/app-init.sh

```diff
@@ -117,29 +115,6 @@ echo "Running initial checks..."
 
 /usr/local/bin/python $WORKSPACEDIR/ietf/manage.py migrate --settings=settings_local
 
-
```

Comment:
We still need one call to migrate in this block - a user might have new migrations in their fork, not yet committed to feat/posgtres that need to be run.

Reply:
Unlikely as its using a new db image?

Reply:
Doh - read right past new L116